gdc_client/upload/parser.py

```diff
@@ -9,15 +9,20 @@
 from . import exceptions
 
 from .client import GDCUploadClient
+from .. import log
 
 
+logger = log.get_logger('upload-client')
+
 def validate_args(parser, args):
     """ Validate argparse namespace.
     """
-    if args.manifest or args.identifier:
+    if args.manifest or args.file_ids:
+        if args.identifier:
```

Comment:
Ahh, yea, I think the user should be warned whenever they use a deprecated flag.  Still not sure I understand what `identifier` did, but will look later